WordPress/Classes/Models/Post.h

```diff
@@ -10,26 +12,26 @@ extern NSString * const PostTypeDefaultIdentifier;
 ///-------------------------------
 /// @name Specific Post properties
 ///-------------------------------
-@property (nonatomic, strong) NSNumber *commentCount;
-@property (nonatomic, strong) NSNumber *likeCount;
-@property (nonatomic, strong) Coordinate *geolocation;
-@property (nonatomic, strong) NSString *tags;
-@property (nonatomic, strong) NSString *postType;
-@property (nonatomic, strong) NSString *postFormat;
-@property (nonatomic, strong) NSString *postFormatText;
-@property (nonatomic, strong) NSSet *categories;
+@property (nonatomic, strong, nullable) NSNumber *commentCount;
+@property (nonatomic, strong, nullable) NSNumber *likeCount;
+@property (nonatomic, strong, nullable) Coordinate *geolocation;
+@property (nonatomic, strong, nullable) NSString *tags;
+@property (nonatomic, strong, nullable) NSString *postType;
```

Comment:
`postType` is actually not an optional value in the CD model, and has a default value. So we could probably remove the `nullable` specifier.